Campaign_Manager/templates/Campaign_Manager/campaign_list.html

```diff
@@ -0,0 +1,25 @@
+{% extends "Campaign_Manager/campaign_base.html" %}
+
+{% block content %}
+    <h1>Current Campaigns</h1>
+
+    {% for campaign in campaigns %}
+
+
+				<!-- Confirmed that user is correctly assigned -->
+				<!-- I would normally put the above in here, but I can't figure out how to
+					assign a user to the character upon first loading so I'll just keep it here -->
+
+				<div>
+					<h2><a href="{% url 'campaign-detail' campaign.campaignID %}">{{ campaign.campaignName }}</a></h2>
```

Comment:
Maybe change the URL to use campaign.campaignName, if possible. I know that will change what you've done in the urls.py, but we would probably rather not expose the primary key of our DB table in the URL. If it's not an easy change, don't worry about it

Reply:
I will discuss this with josh, I kinda just copied what he had for the character list to get this up and running. I agree that we don't want that key in the URL as well, so I will see what we can do.